frontend/src/services/AuthService.ts

```diff
@@ -7,6 +7,8 @@ import {
 } from 'aws-amplify/auth';
 import { env } from '../env';
 import { UserClientRolesType } from '../types/UserRoleType';
```

Comment:
nitpick: I know this is not your change, but this import is unused, can be removed.